test/await-after.test.js

```diff
@@ -280,6 +280,54 @@ test('await after complex scenario', async (t) => {
   }
 })
 
+test('without autostart and sync/async plugin mix', async (t) => {
```

Comment:
Would you mind to add the same test but reading a file using fs instead of the sleep function?

Reply:
Do you mean an FS read within a plugin register function?  or in between registrations? (or both? 😄)

Reply:
I just pushed a commit that places some of the `sleep` calls with FS reads.  Please let me know if this is/isn't what you had in mind.

Reply:
Your update covers all these cases 😄 

Thanks for the check (the timers happens before the IO queue)